src/App.js

```diff
@@ -2,77 +2,50 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
-  _isMounted = false;
```

Comment:
Not sure what this is being used for, didn't seem to play any role, so deleted it